### ISSUE TITLE
the servers don't show the number 

### ISSUE THREAD

**Issue by Duui3111** [Sank6/Discord-Bot-List#129]
the servers don't show the number of servers the bot is in

**Comment by GramingFoxTeam** [comment]
> the servers don't show the number of servers the bot is in

u need to use api on wiki

**Comment by Duui3111** [comment]
ya where do i put it and what is a "YOUR_AUTH_TOKEN"
```js
var myHeaders = new Headers();
myHeaders.append("authorization", "YOUR_AUTH_TOKEN");
myHeaders.append("Content-Type", "application/json");

var requestOptions = {
  method: 'POST',
  headers: myHeaders,
  body: JSON.stringify({"server_count": 1500}); // Replace this number with the server count
};

fetch("/api/auth/stats/:botid", requestOptions) // Make sure you include the domain
  .then(response => response.text())
  .then(console.log)
  .catch(console.error);
```

**Comment by Sank6** [comment]
You can access your auth token from your bot's edit page.

**Comment by Duui3111** [comment]
o ya, but where does the code go

**Comment by Sank6** [comment]
That's posting server count from your bot. So that should be in the same process that you're running your bot client on.

**Comment by Duui3111** [comment]
i don't get it like i the status.js

**Comment by Sank6** [comment]
I'm not sure what you mean?

**Comment by Zaid-maker** [comment]
You can simply use node-fetch for it

**Comment by Duui3111** [comment]
I dont’t get a t

**Comment by Sank6** [comment]
In your bot...

**Comment by Duui3111** [comment]
so in the bot.js file

**Comment by Duui3111** [comment]
i ant finger it what where do i put this

**Comment by Duui3111** [comment]
do you mean in the main bot file because it tells me 
`ReferenceError: Headers is not defined`

**Comment by Duui3111** [comment]
I still cant finger it out

**Comment by Duui3111** [comment]
did you figure out why i get the error

**Comment by Sank6** [comment]
I haven't had a look at the stat posting yet. I'll leave this open till I do.

**Comment by Sank6** [comment]
Fixed: 5738936e3aff0c1399b071410da22e849ae7203c. Sorry for the delay.
